packages/execute-driver-plugin/lib/plugin.js

```diff
@@ -32,7 +32,7 @@ export default class ExecuteDriverPlugin extends BasePlugin {
    *
    * @returns {Promise<any>} - a JSONifiable object representing the return value of
    * the script
-   * @type {import('@appium/types').PluginCommand<[string, string?, number?]>}
+   * @type {import('@appium/types').PluginCommand<[script:string, scriptType:string?, timeoutMs:number?], any>}
```

Comment:
everywhere else we have spaces after `:`, probably should here for consistency

Reply:
yeah, that was on purpose due to a tooling problem (I have a thing that automatically rewraps comment paragraphs which I should probably disable).